lucille-core/src/test/java/com/kmwllc/lucille/indexer/SolrIndexerTest.java

```diff
@@ -387,6 +446,37 @@ public void testIndexerWithChildDocWithNestedJsonWithObjects() throws Exception
       Event.Type.FAIL, equalTo(events.get(0).getType()));
   }
 
+  /**
```

Comment:
I added this test last week. Looks like it's now redundant with testSolrIndexerWithMultipleCollections() that you added. If that's the case, feel free to remove it.

In the future if I'm making changes to a PR I'll let you know beforehand or else do it in a separate PR. Sorry about that. I had sketched this out and thought it would be easier to commit than describe it in a comment.